great_expectations/data_context/data_context/cloud_data_context.py

```diff
@@ -820,11 +821,17 @@ def _persist_suite_with_store(
 
         return expectation_suite
 
-    def _save_project_config(self) -> None:
+    def _save_project_config(
+        self, _fds_datasource: FluentDatasource | None = None
+    ) -> None:
         """
         See parent 'AbstractDataContext._save_project_config()` for more information.
 
         Explicitly override base class implementation to retain legacy behavior.
         """
-        logger.debug(f"{type(self).__name__}._save_project_config() is a NoOp")
-        # TODO: this may be adjusted as part of fluent datasources save flow
+        if _fds_datasource:
+            self._datasource_store.set(key=None, value=_fds_datasource)
```

Comment:
One alternative to this would be to leave the `_save_project_config` signature as it was and iterate over all the `context.fluent_datasources` and set them all.
But that's a lot HTTP requests and is asking for concurrency and local consitency problems.

Reply:
LGTM!